usco/Jam
# Multi selection editing of transformations

As a user I would like to be edit multiple parts at the same time
## TODO: 
- [x] implement multi editing support for 2D ui
- [x] implement multi editing support for 3D ui (3d gizmos)
- [x] implement different selection scheme to make multi selection more convenient 
     - "additive" selection : selecting another part after already selecting the current one should ADD the new part to the list of selections)
     - combining selection : selecting a superset of the current selection should select the superset, but selecting a subset of the current selection should select just that item
- [x] add system using long 'presses/taps' to toggle multi selection
- [x] add small explanation in help



## How to test
- check the behaviour of selections, should match the description above
- single  selection based editing should work as it used to : select a part, change its transforms (position rotation scale)
- multiple selection based editing should work as well, in the same manner as the single selection does
- also check snap & uniform transforms
